test/groovy/com/sap/piper/UtilsTest.groovy

```diff
@@ -76,4 +76,142 @@ class UtilsTest extends BasePiperTest {
         List result = Utils.appendParameterToStringList(['string'], parameters, 'param')
         assertEquals(['string'], result)
     }
+
+    def newExaminee(Map results) {
+        results.stashProperties = null
+        def examinee = new Utils()
+        examinee.steps = [
+            stash: { Map stashProperties ->
+                results.stashProperties = stashProperties
+            },
+        ]
+        examinee.echo = {}
+        return examinee
+    }
+
+    @Test
+    void testStash_noParentheses() {
```

Comment:
a lot of the tests here are similar. We should think about parameterized tests.

Reply:
Had the same thoughts and decided to have separate tests.
With parameterized tests it is harder to identify which one failed (if not good implemented).
And it is not that much lines of code overhead.